requirements.txt

```
@@ -17,6 +17,6 @@ pydantic-settings~=2.3
 pymsteams~=0.2.2   # TODO remove teams dependency
 scikit-learn~=1.3
 scipy~=1.10
-statsmodels~=0.13.5
+statsmodels>=0.13.5
```

Sounds good. I don't see a problem with that, but we should still add some constraint to the versioning in case they release a breaking version.

```suggestion
statsmodels>=0.13.5,<1.0.0
```